cron/hourly/reconcile-transactions.js

```diff
@@ -48,6 +50,32 @@ async function reconcileConnectedAccount(connectedAccount) {
           'approvals',
         );
 
+        const collectiveId = card.collectiveId;
```

Comment:
Isn't this one uppercase?

```suggestion
        const collectiveId = card.CollectiveId;
```